Postman Test Suite/Ed-Fi ODS-API Integration Test Suite ResponseTests.postman_collection.json

```diff
@@ -4834,14 +4819,16 @@
 															"    const scenarioId = pm.environment.get('scenarioId');",
 															"    const knownStudentId = pm.environment.get('known:'+scenarioId+':studentId');",
 															"",
-															"    // Remove the correlationId query string parameter from the request url",
-															"    pm.request.url.removeQueryParams('correlationId');",
+															"    const actualLocation = pm.response.headers.one('location').value;",
+															"    ",
+															"    const expectedResourceUrl = pm.variables.replaceIn(\"{{ApiBaseUrl}}/data/v3/ed-fi/students\");",
```

Comment:
Using the well-known path to the resource rather than a cleaned up variant of the request URL returned by Postman. This produces a stable expected value not dependent on variations on the request (like a trailing `/` on the URL).